azure-media/src/main/java/com/microsoft/windowsazure/services/media/entityoperations/EntityRestProxy.java

```diff
@@ -196,7 +196,7 @@ public String update(EntityUpdateOperation updater) throws ServiceException {
         updater.processResponse(clientResponse);
         if (clientResponse.getHeaders().containsKey("operation-id")) {
             List<String> operationIds = clientResponse.getHeaders().get("operation-id");
-            if (operationIds.size() >= 0) {
```

Comment:
if the operationIds.size is ZERO, then operationIds.get(0) will throw index out of bound exception. Fixing it